### ISSUE TITLE
l2 norm calcuate error

### ISSUE THREAD

**Issue by gneworld** [koyeongmin/PINet_new#12]
https://github.com/koyeongmin/PINet_new/blob/fead6baf0210a11444bf02d8fda7b3a33cec8e8c/CurveLanes/test.py#L439

Is it should be dis = np.linalg.norm((feature[i] - j))?

**Comment by koyeongmin** [comment]
Yes, l2 norm is dis = np.linalg.norm((feature[i] - j)). This is my mistake, but distinguishing each line depends on distance because them, it can work well if you determine suitable threshold. Thank you!
